modules/server/src/network/resolvers/aggregations.ts

```diff
@@ -20,12 +31,15 @@ import { CONNECTION_STATUS, NetworkNode } from './networkNode';
 const fetchData = async <SuccessType>(
 	query: NetworkQuery,
 ): Promise<Result<SuccessType, typeof CONNECTION_STATUS.error>> => {
-	const { url, gqlQuery } = query;
+	const { url, gqlQuery, queryVariables } = query;
+
 	console.log(`Fetch data starting for ${url}`);
```

Comment:
ToDo: consider logger instance/level, so this doesn't come up in regular Arranger instances (unless that may actually be desirable?)